src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/BookingService.kt

```diff
@@ -237,22 +237,44 @@ class BookingService(
       val lengthOfStayWeeks = lengthOfStayDays.toDouble() / 7
       val lengthOfStayWeeksWholeNumber = (lengthOfStayDays.toDouble() % 7) == 0.0
 
-      emailNotificationService.sendEmail(
-        user = applicationSubmittedByUser,
-        templateId = notifyConfig.templates.bookingMade,
-        personalisation = mapOf(
-          "name" to applicationSubmittedByUser.name,
-          "apName" to premises!!.name,
-          "applicationUrl" to applicationUrlTemplate.replace("#id", placementRequest.application.id.toString()),
-          "bookingUrl" to bookingUrlTemplate.replace("#premisesId", booking.premises.id.toString())
-            .replace("#bookingId", booking.id.toString()),
-          "crn" to placementRequest.application.crn,
-          "startDate" to arrivalDate.toString(),
-          "endDate" to departureDate.toString(),
-          "lengthStay" to if (lengthOfStayWeeksWholeNumber) lengthOfStayWeeks else lengthOfStayDays,
-          "lengthStayUnit" to if (lengthOfStayWeeksWholeNumber) "weeks" else "days",
-        ),
-      )
+      if (applicationSubmittedByUser.email != null) {
+        emailNotificationService.sendEmail(
+          email = applicationSubmittedByUser.email!!,
+          templateId = notifyConfig.templates.bookingMade,
+          personalisation = mapOf(
+            "name" to applicationSubmittedByUser.name,
+            "apName" to premises!!.name,
+            "applicationUrl" to applicationUrlTemplate.replace("#id", placementRequest.application.id.toString()),
+            "bookingUrl" to bookingUrlTemplate.replace("#premisesId", booking.premises.id.toString())
+              .replace("#bookingId", booking.id.toString()),
+            "crn" to placementRequest.application.crn,
+            "startDate" to arrivalDate.toString(),
+            "endDate" to departureDate.toString(),
+            "lengthStay" to if (lengthOfStayWeeksWholeNumber) lengthOfStayWeeks else lengthOfStayDays,
+            "lengthStayUnit" to if (lengthOfStayWeeksWholeNumber) "weeks" else "days",
+          ),
+        )
+      }
+      if (premises != null) {
+        if (premises!!.emailAddress != null) {
```

Comment:
I think you can just do `if (premises?.emailAddress != null) {` here rather than the two conditionals